### ISSUE TITLE
worldserver fails to start due to database model issue

### ISSUE THREAD

**Issue by neohyuga** [azerothcore/azerothcore-wotlk#19425]
### Current Behaviour

Fresh compile of core on Linux.
Starting worldserver for the first time and database was being populated.
Eventually core continued to load, but then got this error:

Loading Creature Model Based Info Data...
>> Loaded 24143 Creature Model Based Info in 10 ms

**Loading Creature Custom IDs Config...
Loading Creature Templates...
[1146] Table 'acore_world.creature_template_model' doesn't exist
Your database structure is not up to date. Please make sure you've executed all queries in the sql/updates folders.**

#----------------------------------------------------------------------#
 
>> ABORTED

# Location '/home/acadmin/azerothcore/src/server/database/Database/MySQLConnection.cpp:640'
# Function '_HandleMySQLErrno'
 
#----------------------------------------------------------------------#
Segmentation fault

### Expected Blizzlike Behaviour

Unsure how to update database to latest/correct version. The documentation states that it should be automatic.

### Steps to reproduce the problem

1. Compile
2. Configure
3. Run worldserver

### AC rev. hash/commit

None

### Operating system

Debian 12

**Comment by Kitzunu** [comment]
You are missing db updates so your db is not good. Use auto updater and clean your db

**Comment by neohyuga** [comment]
I thought that running worldserver with the default settings for the database autoupdater would ensure the world database was created and updated properly.

I dropped the world database, then used dbimport.

I saw this error:

>> Applying 'creature_template_model.sql'...
ERROR 1273 (HY000) at line 19 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Unknown collation: 'utf8mb4_0900_ai_ci'
ERROR 1146 (42S02) at line 31 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Table 'acore_world.creature_template_model' doesn't exist
ERROR 1146 (42S02) at line 32 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Table 'acore_world.creature_template_model' doesn't exist
ERROR 1146 (42S02) at line 10033 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Table 'acore_world.creature_template_model' doesn't exist
ERROR 1146 (42S02) at line 20034 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Table 'acore_world.creature_template_model' doesn't exist
ERROR 1146 (42S02) at line 30035 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Table 'acore_world.creature_template_model' doesn't exist
ERROR 1146 (42S02) at line 40036 in file: '/home/acadmin/azerothcore/data/sql/base/db_world/creature_template_model.sql': Table 'acore_world.creature_template_model' doesn't exist


Perhaps my database version has changes that no longer allow that collation?

Regardless, I ended up editing "creature_template_model.sql" and changing the collation to:

COLLATE=utf8mb4_unicode_ci


Then I re-ran dbimport and recreated the world database.

And that worked!

**Comment by Kitzunu** [comment]
What database do you use?

**Comment by Kitzunu** [comment]
You are using maria and somehow that file got fucked, change collation to COLLATE=utf8mb4_unicode_ci and it will work

**Comment by Kitzunu** [comment]
Update to latest master and it will work

**Comment by avarishd** [comment]
When exactly are we cutting the life support of mariadb?

**Comment by Kitzunu** [comment]
When we support MySQL 8.4

**Comment by neohyuga** [comment]
Using:

10.11.6-MariaDB-0+deb12u1 Debian 12

That's the version that was installed when I issued the "apt install" command under the "Debian 10 / Debian 12" heading in the documentation page: https://www.azerothcore.org/wiki/linux-requirements

git cmake make gcc g++ clang default-libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev mariadb-server libboost-all-dev

I did see now that under "Ubuntu with MariaDB 10.x" heading, it says: AzerothCore does only support MariaDB versions 10.6 and 10.5.

Maybe that means that 10.11.6 is not supported.

In any case, I appreciate you time and help with this issue!
